pkg/webrtctransport.go

```diff
@@ -186,7 +190,37 @@ func NewWebRTCTransport(ctx context.Context, session *Session, me MediaEngine, c
 
 // CreateOffer generates the localDescription
 func (p *WebRTCTransport) CreateOffer() (webrtc.SessionDescription, error) {
-	return p.pc.CreateOffer(nil)
+	offer, err := p.pc.CreateOffer(nil)
```

Comment:
Create offer already return a parsed offer, but it's private, I wonder if we should make it public